### DISCUSSION TITLE
Update Check - Option for requests to api.github.com

### DISCUSSION THREAD

**Discussion by reichelper** [xyzroe/XZG#60] [Ideas]
Hi,
thanks for the great work and your development.

May you see a possibility to add an option in the settings so the request-intervall for 
api.github.com could manually be changed  to own conditions?
Actually, the intervall is 60 minutes, I think once a day (1440 minutes) is also enough. 

What are you thinking about?

**Comment by xyzroe** [comment]
Good idea. I'm planning to rework the update availability check (cause it takes place into your browser using JS and on ESP32). I'll make an option to set this interval vis UI.
